packages/calypso-e2e/src/lib/pages/cart-checkout-page.ts

```diff
@@ -61,12 +61,12 @@ const selectors = {
 	couponCodeInput: `input[id="order-review-coupon"]`,
 	couponCodeApplyButton: `button:text("Apply")`,
 	disabledButton: 'button[disabled]:has-text("Processing")',
-	paymentButton: `button.checkout-button`,
+	paymentButton: `.checkout-submit-button button`,
 	totalAmount:
 		envVariables.VIEWPORT_NAME === 'mobile'
 			? '.wp-checkout__total-price'
 			: '.wp-checkout-order-summary__total-price',
-	purchaseButton: `button.checkout-button:has-text("Pay")`,
+	purchaseButton: `.checkout-submit-button button:has-text("Pay")`,
```

Comment:
Do you need to change this at all if the classes on the buttons aren't changing? It's not a problem but I'm just curious. Because if we don't need to change it then there's no chance of breaking the e2e tests accidentally.

Reply:
Hmm, good question. The new selector is a bit more specific as shown here:

![image](https://github.com/Automattic/wp-calypso/assets/16580129/a56da129-2c3e-4398-b0bc-19413f47568d)

The former `button.checkout-button` will check all buttons for the "Pay" text, including edit and remove buttons, where the latter `.checkout-submit-button button` will only check the payment buttons. In practice it doesn't appear to matter, but I suppose being more specific couldn't hurt and may ward off future issues 🤔

Reply:
Sounds fine to me.